### PR TITLE
Removing oneiric as allowed series.

### PR DESCRIPTION
**Pull request by urosj**

### CODE REVIEW

**Comment by bac** [comment]
:+1:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/483/

File: internal/router/router.go [hunk 1]
```diff
@@ -30,7 +30,6 @@ import (
 
 var knownSeries = map[string]bool{
 	"bundle":  true,
-	"oneiric": true,
 	"precise": true,
 	"quantal": true,
 	"raring":  true,
```
Comment (jrwren) [inline internal/router/router.go]: raring and saucy are no longer supported either. Should they be removed?

**Comment by jrwren** [comment]
:+1: with comment.

**Comment by frankban** [comment]
:+1: after removing quantal/raring and adding utopic.
Thank you!

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results: http://ci.jujugui.org:8080/job/charmstore/484/

**Comment by urosj** [comment]
:shipit:

**Pull request merged**